foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -66,6 +66,21 @@ In practice you will likely not be using complex values for `flex-grow`, `flex-s
 
 It _is_ possible to get fancy, and set up layouts where some columns relate to each other in a specific ratio, so it's useful to know that you can use other values, but those are relatively rare.
 
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. Read <a href ="https://www.w3.org/TR/css-flexbox-1/#flex-common">this section</a> to understand the basic values of common flex shorthand values
+
+2. This <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/flex">MDN Doc</a> summarizes the entire flex short hand values as well as introduces some new syntax that hasn't been covered in the previous 2 articles.
+
+3. This <a href="https://css-tricks.com/understanding-flex-grow-flex-shrink-and-flex-basis/">CSS Tricks</a> article that explains the individual flex-grow, flex-shrink and flex-basis in detail
+
+4. Finally, Watch this <a href="https://www.youtube.com/watch?v=vh-ZpSIcwI8">video</a> to help summarize the concepts of flex-grow, flex-shrink, and flex basis that were covered in the previous article.
```

Comment:
**issue:** I have some concerns about this particular video. I appreciate it demoing some options with `flex-grow`, `flex-shrink`, `flex-basis` and `flex` shorthand, but I don't believe it covers `flex: 1` or how things like the `flex-basis` default value are different when using something like `flex: <positive-number>` which has been the basis of confusion in this lesson. The YouTuber also seems to get a bit confused at times and I am not confident in this as an Assignment resource. 

**suggestion**: I suggest we simply remove the video. If there is an alternative we could replace it but I don't think a video is necessary unless something was very well explained.

Reply:
Got it, If I come another alternative I will update this however for now I will remove it as said

Thanks for the input!